packages/mainmenu-extension/src/index.ts

```diff
@@ -383,6 +391,14 @@ export function createFileMenu(
     }
   });
 
+  commands.addCommand(CommandIDs.logout, {
+    label: 'Logout',
+    caption: 'Log out of JupyterLab',
```

Comment:
`title: 'Logout confirmation'`,

Reply:
I don't have a confirmation dialog for logout, should I add one? The classic notebook didn't

Reply:
Ah, that's the difference, we don't need a title because we don't have a dialog? Okay, never mind. Let's go with the classic behavior for now, unless there is a strong reason to change.